packages/medusa/src/services/product-variant.ts

```diff
@@ -295,6 +299,7 @@ class ProductVariantService extends TransactionBaseService {
 
       if (prices) {
         await this.updateVariantPrices(variant.id!, prices)
+        await this.cacheService_.invalidate(`ps:${variant.id}:*`)
```

Comment:
Yes, that's true @dwene.

I think it's a good suggestion to add a method that is called after a variant update to the `PriceSelectionStrategy` and invalidate the cache there. Let's see what @olivermrbl and @pKorsholm will say.

Reply:
I think it's a great point to add it considering, as @dwene pointed out it would be great to give control of the invalidation to the developers of the strategy. 
No objections from my side 😄

Reply:
This is excellent input @dwene - thanks.

I like your suggestion of allowing multiple variants to be marked as updated at the same time. I suggest we expose the following method in the strategy:

```
async onVariantsPricesUpdate(variantIds: string[]): Promise<void>
```

Reply:
Sounds good - added the method in https://github.com/medusajs/medusa/pull/3553/commits/aca5acb9f93c5dbd6688d134ce8de63d1e9f22cb